README.md

```diff
@@ -137,7 +140,106 @@ function debug(plugins) {
 }
 ```
 
-##### `lint`
+#### `encapsulate`
+
+* Type: `Boolean`
+* Default: `false`
+
+Resets CSS properties to their default values to effectively allow a component
+to opt out of CSS inheritance and be encapsulated from the rest of the
+application similar to [the Shadow DOM](https://www.w3.org/TR/shadow-dom/).
+There are two types of CSS properties that affect components, inherited (e.g.
+`font-size,` `color`) and non-inherited (e.g. `margin`, `background`). This
+option works so that:
+
+* Root elements (e.g. `.Component`) have both inherited and non-inherited
+  properties reset to default values.
+* Descendants (e.g. `.Component-item`) only have non-inherited properties reset
+  as this allows properties set on the root element to be inherited by its
+  descendants.
+
+This means that components are isolated from styles outside the component root
+element but should an inheritable property such as `font-size` be applied on the
+component root element it will be inherited by the component descendants as
+normal. The same rules apply to nested components.
+
+##### Why not use `all: initial`?
+
+The `all: initial` declaration will reset even the User Agent styles which is
+often too heavy handed:
+
+> For example, if an author specifies `all: initial` on an element it will block
+  all inheritance and reset all properties, as if no rules appeared in the
+  author, user, or user-agent levels of the cascade.
+  This can be useful for the root element of a "widget" included in a page, which
+  does not wish to inherit the styles of the outer page. Note, however, that any
+  "default" style applied to that element (such as, e.g. `display: block` from the
+  UA style sheet on block elements such as `<div>`) will also be blown away.
+
+https://www.w3.org/TR/css3-cascade/#all-shorthand
+
+Instead the preprocessor uses [postcss-autoreset](https://github.com/maximkoretskiy/postcss-autoreset)
+with the SUIT preset and a custom set of CSS properties that are reset to their
+default values. Only selectors conforming to the SUIT naming conventions are effected.
```

Comment:
> reset to their initial values

default could be interpreted as the UA defaults eg. blue and underlined for links